src/Features/CSharp/Portable/Completion/SuggestionMode/CSharpSuggestionModeCompletionProvider.cs

```diff
@@ -91,6 +93,20 @@ private bool IsAnonymousObjectCreation(SyntaxToken token)
             return false;
         }
 
+        private bool IsPossibleTupleExpression(SyntaxToken token)
```

Comment:
can we make any smarter decisions if this is an actual tuple expression or not?  For example, we do similar 'builder' work when we think something could be a simple lambda.  However, we do *not* say "oh, we say an identifier, so that might be the start of a lambda".  Instead we say "oh, we say an identifier, and in this context it seems likely that a lambda could appear, so *now* we'll use a builder".

This check seems very broad, making it so that after any open paren we don't show things.  However, if i write:  ```Console.WriteLine($$```, i would not expect a  builder here.

Also, i would want to check the semantics of that location to see if a tuple type was likely to occur.  i.e. if i call ```RandomMethod((``` it would not be a tuple locatoin.  But if i call ```ITakeATuple((```, then we would think this was a tuple creation location.